### PR TITLE
Fix an issue that container/sandbox can't be stopped.

### PR DESCRIPTION
**Pull request by Random-Liu**
Today, I found that some containers on my workstation can't be stopped with `crictl stop`.

It turns out that the container processes are moved to `/user.slice` from the original container cgroups. I guess it is because I simply run `./containerd` to test containerd, without setting `Delegate=true` on my systemd node.
```console
$ /usr/local/bin/ctr -n=k8s.io task ls | grep 206190
6f757ff0e1c2b544da02c3ab17488be73b843dcc780847ae32ca4ae756be936f    206190    RUNNING
$ cat /sys/fs/cgroup/cpu,cpuacct/user.slice/cgroup.procs | grep 206190
206190
$ cat /sys/fs/cgroup/cpu,cpuacct/kubepods/burstable/podcedaeae7-a4e4-11e8-a6f6-8cdcd43ac064/6f757ff0e1c2b544da02c3ab17488be73b843dcc780847ae32ca4ae756be936f/cgroup.procs
```

Although what I did was not best practice, I still think we should handle this case, not being able to stop container is very bad.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

**Comment by Random-Liu** [comment]
Should cherrypick into all supported branches.

File: pkg/server/container_stop.go [hunk 1]
```diff
@@ -81,52 +94,47 @@ func (c *criService) stopContainer(ctx context.Context, container containerstore
...
-	// Event handler will Delete the container from containerd after it handles the Exited event.
-	logrus.Infof("Kill container %q", id)
-	if task != nil {
-		if err = task.Kill(ctx, unix.SIGKILL, containerd.WithKillAll); err != nil {
-			if !errdefs.IsNotFound(err) {
-				return errors.Wrapf(err, "failed to kill container %q", id)
-			}
-			// Move on to make sure container status is updated.
-		}
+	logrus.WithError(err).Errorf("Kill container %q timed out", id)
```
Comment (mikebrow) [inline pkg/server/container_stop.go]: waitContainerStop() has other error types.. should probably use "an error occurs during waiting for container %q to stop"

Comment (Random-Liu) [reply pkg/server/container_stop.go]: Make sense.

Comment (Random-Liu) [reply pkg/server/container_stop.go]: Done

**Comment by Random-Liu** [comment]
@mikebrow Addressed comments. Please take another look. :)

**Pull request merged**